src/index.js

```diff
@@ -11,6 +12,11 @@ ReactDOM.render(
   document.getElementById('root')
 );
 
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://cra.link/PWA
+serviceWorkerRegistration.register();
```

Comment:
This is where we are going to need to add a bunch of boilerplate logic for notifications